src/hub/Accounting.sol

```diff
@@ -63,7 +63,7 @@ contract Accounting is Auth, IAccounting {
         debited = 0;
         credited = 0;
         _currentPoolId = poolId;
-        
+
         if (TransientJournal.journalId(poolId) == 0) {
             TransientJournal.setJournalId(poolId, _generateJournalId(poolId));
         }
```

Comment:
TODO: remove this. it seems unused: https://kflabs.slack.com/archives/C07PG2EUR9C/p1744712036752739

Reply:
Unless we still want to link message in batches using the JournalId, not sure if it's still a feature